samples/apps/txregulator/tests/txregulatorclient.py

```diff
@@ -270,7 +270,7 @@ def run(args):
                             c.rpc("REG_get_revealed", {"tx_id": tx_id}),
                             error=lambda e: e is not None
                             and e["code"]
-                            == infra.jsonrpc.ErrorCode.INVALID_PARAMS.value,
+                            == infra.client.ErrorCode.INVALID_PARAMS.value,
```

Comment:
I don't think these error codes should be moved into `infra.client` - they're JSON RPC specific, so ideally they'd stay under `infra.jsonrpc`.